lib/veri_traci.ml

```diff
@@ -0,0 +1,89 @@
+open Bap.Std
+module Bap_value = Value
+open Bap_traces.Std
+open Monad.State
+open Bil.Result
+
+type event = Trace.event
+
+let mem_of_arch arch = 
+  let (module T : Target) = target_of_arch arch in
+  T.CPU.mem
+
+(** TODO: what is a behaviour of update_event method?  *)
+class context arch = object(self:'s)
+  inherit Bili.context
+  val mem  : var = mem_of_arch arch
+  val arch : arch = arch
+  method arch = arch
+  method mem = mem
+  method update_event: event -> 's = fun ev -> failwith "unimplemented"
+end
+
+let mem_operands mv = 
+  Size.of_int_opt (Bitvector.bitwidth (Move.data mv)) |> function
```

Comment:
Use module `Word` instead of bitvector. Or `Addr` if you're working with addresses